pkg/auth/auth.go

```diff
@@ -267,3 +283,21 @@ func (u *UserAuth) GetForbiddenActions(namespace, action, manifest string) ([]Ac
 	}
 	return forbiddenActions, nil
 }
+
+// ParseForbiddenActions parses a forbidden error returned by the Kubernetes API and return the list of forbidden actions
+func ParseForbiddenActions(message string) []Action {
+	re := regexp.MustCompile(`User "(.*?)" cannot (.*?) resource "(.*?)" in API group "(.*?)"(?: in the namespace "(.*?)")?`)
```

Comment:
Please include a big TODO to remove this with a link to the helm bug, or similar?

Reply:
I think we meant different things with what we expect from Helm. My idea is for them to return all the elements that fails when creating a release (not just the first) but we would still need to parse this message with a regexp. 

Apparently, you are thinking on completely removing this regexp but what would replace it?

Reply:
Yes, I assumed you meant both that helm would return all the elements that fail and that it would do so in a structured way rather than as a big string message. With the new error package it's pretty easy to have an error type which includes a json representation of the error in addition to the textual string, for example. (Or even without the new errors functionality - it's just neater using `errors.As` or `errors.IsA` - the wrapping itself wouldn't be necessary).

Reply:
okay, noted but even if they do that I think some regex will be needed.